daemon/state.go

```diff
@@ -346,6 +346,14 @@ func (d *Daemon) initRestore(restoredEndpoints *endpointRestoreState) chan struc
 
 		go func() {
 			if k8s.IsEnabled() {
+				// Also wait for all cluster mesh to be synchronized with the
+				// datapath before proceeding.
+				if option.Config.ClusterID != 0 {
```

Comment:
Maybe more explicit option (e.g. as we have for k8s) to detect whether cluster-mesh is enabled?

Reply:
There's no such option